src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitStorage = new HashMap<>();
+
+    public static void setBalance(String fruit, int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("quantity can't be negative");
+        }
+        fruitStorage.put(fruit, quantity);
+    }
+
+    public static void addFruit(String fruit, int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("quantity can't be negative");
+        }
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public static void removeFruit(String fruit, int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("quantity can't be negative");
+        }
+        fruitStorage.put(fruit, fruitStorage.getOrDefault(fruit, 0) - quantity);
+    }
+
+    public static Map<String, Integer> getAllFruits() {
```

Comment:
The `getAllFruits` method is exposing the internal storage map directly, which breaks encapsulation. Instead, consider returning a read-only view of the storage or a copy of the map to prevent external modifications.